### ISSUE TITLE
Count character 'n' for NPOL SMILES

### ISSUE THREAD

**Issue by alvaroingold** [alvaroing12/atlas-cbs-ipm#1]
Bug in the 'on the fly' counting of NPOL, when the SMILES contain a character 'n' (lower case N), meaning that they are included in aromatic rings. This needs to be corrected, as this problem is very prominent whe using the import CSV, SMI feature

**Comment by alvaroingold** [comment]
Fixing to count the lower case 'n' carried out with SMARTS. According to Daylight examples:

```
Acceptors are defined as nitrogen or oxygen:
[N,n,O,o] or [#7,#8]
```

The current code implements the following parsing strategy for SMARTSQueryTool at classes 
**1. LEIaddCompound**
**2. LEIupdateCompound**
**3. LEIuploadSMI**
**4. LEIuploadCSV**

```
==> SMARTSQueryTool querytool = new SMARTSQueryTool("[N,n,O,o]");
boolean status = querytool.matches(ac);
if (status) { npol = querytool.countMatches(); }                                                                        
```

**Comment by alvaroingold** [comment]
Issue fixed by commit 4de8eaaaa180da7855886601c8338450a6448a23
